content/partials/quickstart/publishing-app-store.md

```diff
@@ -23,10 +23,15 @@ publishing:
     # Configuration related to TestFlight (optional)
 
     # Optional boolean, defaults to false. Whether or not to submit the uploaded
-    # build to TestFlight beta review. Required for distributing to beta groups.
+    # build to TestFlight beta review. Required for distributing to beta groups and expiring previous builds.
     # Note: This action is performed during post-processing.
     submit_to_testflight: true 
 
+    # Optional boolean, defaults to false. Whether to expire previous builds uploaded
```

Comment:
Added more info, but no, there is no such thing as "status" for builds themselves (other than expired/not expired), so it can not be specified. Apple API does not allow directly modifying a beta review submission (canceling) - thus, the "rejection" is done via expiring builds.